data/gtk-style.css

```diff
@@ -484,7 +484,7 @@ ActivityCard {
         background: #3f3f3f; }
       ActivityCard #bottombox #play.running:hover {
         background: #262626; }
-      ActivityCard #bottombox #play:disabled, ActivityCard #bottombox #play.complete:disabled {
+      ActivityCard #bottombox #play:disabled, ActivityCard #bottombox #play:disabled.complete {
```

Comment:
The SASS compilation keeps switching the order every time. We need to do something about this. For now, please remove it manually from the commit, as this is messing the git history.

Reply:
Yes, I didn't notice this change, I've removed by hand so the diff shows the real changes.